addons/netfox/network-time.gd

```diff
@@ -196,7 +196,7 @@ var ticktime: float:
 var tick_factor: float:
 	get:
 		if not sync_to_physics:
-			return 1.0 - clampf(_next_tick * tickrate, 0, 1)
+			return 1.0 - clampf((_next_tick - _get_os_time()) * tickrate, 0, 1)
```

Comment:
The tick interpolators jitter as time passes by. The longer a process loop takes the more pronounced the jitter.

We need to pin the time at the start of the process loop and base interpolation on that.


```suggestion
			return 1.0 - clampf((_next_tick - _current_tick) * tickrate, 0, 1)
```